### PR TITLE
INT-5556 Add no-console lint rule

### PR DESCRIPTION
**Pull request by edreyer1**

### CODE REVIEW

File: packages/integration-sdk-dev-tools/config/eslint.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
   ],
   "rules": {
     "no-unused-vars": "off",
+    "no-console": "error",
```
Comment (ndowmon) [inline packages/integration-sdk-dev-tools/config/eslint.json:50]: Have you confirmed (perhaps by linking `@jupiterone/integration-sdk-dev-tools` in a local version of `graph-aws`) that the addition of this rule causes `yarn lint` to fail if you add a `console.log()` statement somewhere in `graph-aws`?

**Review by ndowmon — APPROVED** [review]

File: packages/integration-sdk-dev-tools/config/eslint.json (hunk 1 above)
Comment (edreyer1) [reply packages/integration-sdk-dev-tools/config/eslint.json:50]: Screen Shot 2022-09-21 at 10 21 54 AM

Comment (edreyer1) [reply packages/integration-sdk-dev-tools/config/eslint.json:50]: Just checked  and yes it does seem to catch console.log

**Review by austinkelleher — APPROVED** [review]
This is awesome. Should we technically do a major version bump for this?

**Pull request merged**